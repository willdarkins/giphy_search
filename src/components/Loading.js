import React from 'react'

const Loading = () => {
  return (
    <div className='loader'>
        <i className='fas fa-spinner fa-4x fa-spin' />
    </div>
  )
}

export default Loading