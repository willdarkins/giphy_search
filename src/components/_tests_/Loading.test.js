import { render, cleanup } from '@testing-library/react';
import Loading from '../Loading'

afterEach(cleanup);

describe('Giphy component', () => {
    it('renders', () => {
      render(<Loading />);
    });
    it('matches snapshot DOM node structure', () => {
        const { asFragment } = render(<Loading />);
        expect(asFragment()).toMatchSnapshot();
    })
  })