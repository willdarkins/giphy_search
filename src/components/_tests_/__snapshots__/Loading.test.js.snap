// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Giphy component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <div
    class="loader"
  >
    <i
      class="fas fa-spinner fa-4x fa-spin"
    />
  </div>
</DocumentFragment>
`;

exports[`Loading component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <div
    class="loader"
  >
    <i
      class="fas fa-spinner fa-4x fa-spin"
    />
  </div>
</DocumentFragment>
`;
